app/src/main/java/com/huji/foodtricks/foodtricks/Recipe.java

```diff
@@ -0,0 +1,49 @@
+package com.huji.foodtricks.foodtricks;
+
+import com.google.gson.JsonObject;
+
+import org.json.JSONException;
+
+import java.io.IOException;
+public class Recipe {
+
+
+    String _name;
+    String _pic_url;
+    String _url;
+    int _duration; // Cooking time in minutes
+
+    public Recipe(JsonObject recipe_json) {
```

Comment:
It would be a good idea to parse the number of ingredients as well, so we can use it to assess the difficulty of a recipe

Reply:
Added to recipe a field called _difficulty